test/e2e/common/runtime.go

```diff
@@ -278,12 +279,17 @@ while true; do sleep 1; done
 					waiting:     false,
 				},
 				{
-					// TODO(claudiub): Remove the [LinuxOnly] tag when a Windows-friendly image is used instead of alpine.
 					description: "should be able to pull image from docker hub [LinuxOnly]",
 					image:       "alpine:3.7",
 					phase:       v1.PodRunning,
 					waiting:     false,
 				},
+				{
+					description: "should be able to pull image from docker hub [WindowsOnly]",
+					image:       "claudiubelu/busybox:1.29",
```

Comment:
so i guess we need to hold this PR until:
- we can have this image in a official account somewhere?
- add an [WindowsOnly] alternative to the test case "should be able to pull image from gcr.io [LinuxOnly]","

is this correct?

Reply:
Regarding the official account, the ``e2eteam`` repo is what we've created to be used by CIs and they are actively maintained and kept up-to-date, so I think that's a good enough.

Regarding the 2nd point, We could push the busybox image to gcr.io as well, I can look into it, but that could also be done in a separate PR.

Updated the repo name to ``e2eteam``.

Reply:
> that could also be done in a separate PR.

ok, agreed.

could you please add TODO about the GCR test for Windows?

Reply:
Done.